pyblazing/pyblazing/apiv2/context.py

```diff
@@ -2065,7 +2082,8 @@ def create_table(self, table_name, input, **kwargs):
                 metadata_ids = table.metadata[
                     ["file_handle_index", "row_group_index"]
                 ].to_pandas()
-                grouped = metadata_ids.groupby("file_handle_index")
+
+                grouped = metadata_ids.groupby(metadata_ids["file_handle_index"])
```

Comment:
This does not make sense. Can you explain this? This used to be groupby a column name, now its group by a column of values?

Reply:
They were remains of code that I did not remove well, good catch thanks!